hbase-thrift/src/main/resources/org/apache/hadoop/hbase/thrift2/hbase.thrift

```diff
@@ -940,6 +940,8 @@ service THBaseService {
    * region gets splitted, the api may return false.
    *
    * @return true if table is available, false if not
+   *
+   * @deprecated since version 2.1.4
```

Comment:
Should be 2.3.0 or 2.2.0? And please add comment to say when will we remove it, I would like to remove it in 3.0.0, to align with the Table interface. So here we need to tell user that we will remove it in 3.0.0 for aligning with the Table interface, as in general, it should be removed in 4.0.0.